test/certifications.js

```diff
@@ -0,0 +1,183 @@
+                expect(res.statusText).to.equal('OK');
+                expect(res.data).to.have.jsonSchema(schema);
+                certificationID = res.data.data[0].id;
+                certificationID2 = res.data.data[1].id;
+            })
+    );
+    it('should be able to get all certifications', () =>
+        certifications.getCertifications()
+            .then((res) => {
+                expect(res.status).to.equal(200);
+                expect(res.statusText).to.equal('OK');
+                expect(res.data).to.have.jsonSchema(schema);
+            })
+    );
+    it('should be able to get a specific certifiation', () =>
+        certifications.getCertification(certificationID)
+            .then((res) => {
+                expect(res.status).to.equal(200);
+                expect(res.statusText).to.equal('OK');
+                expect(res.data).to.have.jsonSchema(schema);
+            })
+    );
+    it('should be able to edit a specific certification', () =>
+        certifications.updateCertification(certificationID, {
+            description: randString.substr(0, 8),
+            title: randString.substr(0, 8),
+            type: 'ASSIGNED_CERT',
+            state: 'ACTIVE'
+        })
+            .then((res) => {
+                expect(res.status).to.equal(200);
+                expect(res.statusText).to.equal('OK');
+                expect(res.data).to.have.jsonSchema(schema);
+            })
+    );
+    it('should be able to remove a certification from a customer', () =>
+        certifications.updateCustomerCertifications(organizationID, customerID, {
+            action: 'remove',
+            certification_ids: [
+                certificationID
+            ]
+        })
+            .then((res) => {
+                expect(res.status).to.equal(200);
+                expect(res.statusText).to.equal('OK');
+                // expect(res.data).to.have.jsonSchema(schema);
+            })
+    );
+    it('should be able to add a certification to a customer', () =>
+        certifications.updateCustomerCertifications(organizationID, customerID, {
+            action: 'add',
+            certification_ids: [
+                certificationID
+            ]
+        })
+            .then((res) => {
+                expect(res.status).to.equal(200);
+                expect(res.statusText).to.equal('OK');
+                // expect(res.data).to.have.jsonSchema(schema);
+            })
+    );
+    it('should be able to get a customers certifications', () =>
+        certifications.getCustomerCertifications(organizationID, customerID)
+            .then((res) => {
+                expect(res.status).to.equal(200);
+                expect(res.statusText).to.equal('OK');
+                expect(res.data).to.have.jsonSchema(schema);
+            })
+    );
+    it('should be create certifications for an organization', () =>
+        certifications.createOrganizationCertifications(organizationID, {
+            certifications: [
```

Comment:
Also I think we could simplify the payload param to just be an array, we don't need to have an object that has just { certifications: } then the actual  data.